configs/default/FOXE-FOXEERF745V3.config

```diff
@@ -0,0 +1,106 @@
+# Betaflight / STM32F745 (S745) 4.1.0 Sep 25 2020 / 01:23:16 (064ca9b75) MSP API: 1.42
+
+board_name FOXEERF745V3_AIO
```

Comment:
The name of the file has to reflect the MFR - board_name

Reply:
corrected it, thank you